server/lib/pbench/__init__.py

```diff
@@ -153,16 +154,25 @@ def get_pbench_logger(caller, config):
         except FileExistsError:
             # directory already exists, ignore
             pass
-        fh = logging.FileHandler(os.path.join(logdir, "{}.log".format(caller)))
-        fh.setLevel(logging.DEBUG)
+
+        if config.logger_type == "file":
+            handler = logging.FileHandler(os.path.join(logdir, "{}.log".format(caller)))
+        elif config.logger_type == "devlog":
+            handler = logging.handlers.SysLogHandler(address='/dev/log')
```

Comment:
If we import `handlers` as I suggested above:

      from logging import handlers

then this becomes:

     handler = handlers.SysLogHandler(....)